dbt/include/mssql/macros/indexes.sql

```diff
@@ -0,0 +1,76 @@
+{% macro drop_all_indexes_on_table() %}
```

Comment:
This adds nothing to the debug log. Each step should produce a note for the logger so that users can see what's happening.

Reply:
So true!  Didn't even think about that.  Will do.  May be a few days before I get to it.